transpile.cf.js

```diff
@@ -1,38 +1,47 @@
 import fs from 'fs'
 import path from 'path'
 
-const empty = x => fs.readdirSync(x).forEach(f => fs.unlinkSync(path.join(x, f)))
-    , ensureEmpty = x => !fs.existsSync(x) ? fs.mkdirSync(x) : empty(x)
+const empty = (x) =>
+      fs.readdirSync(x).forEach((f) => fs.unlinkSync(path.join(x, f)))
+    , ensureEmpty = (x) => (!fs.existsSync(x) ? fs.mkdirSync(x) : empty(x))
     , root = 'cf'
     , src = path.join(root, 'src')
 
 ensureEmpty(src)
 
-fs.readdirSync('src').forEach(name =>
+fs.readdirSync('src').forEach((name) =>
   fs.writeFileSync(
     path.join(src, name),
     transpile(fs.readFileSync(path.join('src', name), 'utf8'), name, 'src')
   )
 )
 
 function transpile(x) {
-  const timers = x.includes('setImmediate')
-    ? 'import { setImmediate, clearImmediate } from \'../polyfills.js\'\n'
-    : ''
-
-  const process = x.includes('process.')
-    ? 'import { process } from \'../polyfills.js\'\n'
-    : ''
+  const polyfills = [
```

Comment:
Reworked polyfill.cf.js to allow for a single ../polyfills.js import (eslint was not happy about the multiple imports)